v23.1/architecture/replication-layer.md

```diff
@@ -108,6 +108,20 @@ Each replica can be "snapshotted", which copies all of its data as of a specific
 
 After loading the snapshot, the node gets up to date by replaying all actions from the Raft group's log that have occurred since the snapshot was taken.
 
+CockroachDB clusters running v23.1 and later can send _delegated snapshots_.  Delegated snapshots can be sent by a Raft follower instead of the leader of a range.  Which follower is chosen depends on the locality of the follower being nearest the replica that is the final recipient of the snapshot. If the follower is not able to send the snapshotf quickly, the attempt is cancelled and the Raft leader sends the snapshot instead. If the follower is not able to send a snapshot that will be valid for the recipient, the request is rerouted to the leader.
```

Comment:
Change "Delegated snapshots can be sent by a Raft follower instead of the leader of a range." to
"Delegated snapshots can be sent by a Raft follower on behalf of the leader of a range."

Typo snapshotf -> snapshot

Reply:
Updated that line, and fixed the typo!  (note to self: spellcheckkkk)